src/main/java/com/sap/cloudfoundry/client/facade/adapters/RawCloudRoute.java

```diff
@@ -15,22 +11,18 @@
 @Value.Immutable
 public abstract class RawCloudRoute extends RawCloudEntity<CloudRoute> {
 
-    public abstract Resource<RouteEntity> getResource();
-
-    public abstract List<Resource<RouteMappingEntity>> getRouteMappingResources();
+    public abstract Route getRoute();
 
     public abstract Derivable<CloudDomain> getDomain();
 
     @Override
     public CloudRoute derive() {
-        Resource<RouteEntity> resource = getResource();
-        RouteEntity entity = resource.getEntity();
+        Route route = getRoute();
         return ImmutableCloudRoute.builder()
-                                  .metadata(parseResourceMetadata(resource))
-                                  .appsUsingRoute(getRouteMappingResources().size())
-                                  .host(entity.getHost())
+                                  .metadata(parseResourceMetadata(route))
+                                  .appsUsingRoute(route.getDestinations().size())
+                                  .host(route.getHost())
```

Comment:
In v3 route object is included "path" parameter of route, so we can set in CloudRoute object.